examples/cassandra/README.md

```diff
@@ -53,54 +53,48 @@ This is a somewhat long tutorial.  If you want to jump straight to the "do it no
 In Kubernetes, the atomic unit of an application is a [_Pod_](../../docs/user-guide/pods.md).  A Pod is one or more containers that _must_ be scheduled onto the same host.  All containers in a pod share a network namespace, and may optionally share mounted volumes.
 In this simple case, we define a single container running Cassandra for our pod:
 
-<!-- BEGIN MUNGE: EXAMPLE cassandra-controller.yaml -->
+<!-- BEGIN MUNGE: EXAMPLE cassandra.yaml -->
 
 ```yaml
 apiVersion: v1
-kind: ReplicationController
+kind: Pod
```

Comment:
@bgrant0607 yes I did and I am familiar with the restrictions about `Pods`, but I think this is somewhat orthogonal. I didn't create the example, I am just fixing the inconsistencies according to an existing issue #19098.

It seems there are ~24 other examples that use a Pod that so I'd suggest opening a different issue about this.

In any case, the change is appropriate as it just makes the readme instructions and the example code consistent with each other, again I didn't create the example in the first place. But if you prefer I can just remove the all Pod part, only keep the `ReplicationController` example and update the readme accordingly.